.github/workflows/master_build_merge.yml

```diff
@@ -390,9 +390,17 @@ jobs:
             FILE_NAME="$3"
             requiredInputs="1 2 3"
             validateInputs "$requiredInputs" "$@" 
-            curl -v -u ${{env.NEXUS_USERNAME}}:${{env.NEXUS_PASSWORD}} \
-                  --upload-file $FILE_PATH \
-                  ${{env.NEXUS_PROTOCOL}}://${{env.NEXUS_SERVER_ENDPOINT}}/repository/${{env.NEXUS_ARTIFACT_REPO}}/${{env.LABEL}}/$GROUP/$FILE_NAME
+            RESPONSE=$(curl -v -u ${{env.NEXUS_USERNAME}}:${{env.NEXUS_PASSWORD}} \
```

Comment:
Create a common function `upload_to_nexus_with_status_check` (or similar name) 
Inputs will be URL to upload and expected status. 

Alternately since 201 is the expected status, keep input as only URL.